packages/liferay-npm-scripts/src/utils/withTempFile.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const fs = require('fs-extra');
+const path = require('path');
+
+const SignalHandler = require('../utils/SignalHandler');
+const getMergedConfig = require('./getMergedConfig');
+
+const BUILD_CONFIG = getMergedConfig('npmscripts', 'build');
+
+/**
+ * Execute a block of code after writing a temporary file. The function ensures
+ * that the temporary file is deleted after the block execution (which is
+ * assumed to be synchronous).
+ *
+ * @param {string} filename
+ * The name of the file.
+ *
+ * @param {string|Buffer} content
+ * Content of the file.
+ *
+ * Note that the content is used as a seed to derive an HEX string to be used as
+ * suffix to avoid collisions.
```

Comment:
This comment seems untrue.

Reply:
Yes. It's a leftover of a previous version :sweat_smile: